src/component/ModalForm.jsx

```diff
@@ -0,0 +1,115 @@
+import * as React from "react";
+import Backdrop from "@mui/material/Backdrop";
+import Box from "@mui/material/Box";
+import Modal from "@mui/material/Modal";
+import Fade from "@mui/material/Fade";
+import Typography from "@mui/material/Typography";
+import { Button, Divider} from "@mui/material";
+import { Formik, Form, Field, ErrorMessage } from "formik";
+import * as Yup from "yup";
+
+const validationSchema = Yup.object().shape({
+  name: Yup.string()
+    .min(5, "Name must be at least 5 characters")
+    .max(30, "Name cannot exceed 30 characters")
+    .required("Name is required"),
+  email: Yup.string()
+    .email("Invalid email address")
+    .required("Email is required"),
+});
+
+const initialValues = {
+  name: "",
+  email: "",
+};
+const style = {
+  position: "absolute",
+  top: "50%",
+  left: "50%",
+  transform: "translate(-50%, -50%)",
+  width: 400,
+  bgcolor: "background.paper",
+  border: "2px solid #00acc1",
+  boxShadow: 24,
+  p: 4,
+};
+
+export default function TransitionsModal({ handleClose, open }) {
```

Comment:
`open ` is a boolean so we should add a prefix to it like ( is, has, can, ... etc)
`handleClose `is called a callback function, so when receiving a function let's put "on" with the event name.
```suggestion
export default function TransitionsModal({ onClose, isOpen }) {
```